src/pages/ocpDashboard/ocpDashboard.tsx

```diff
@@ -51,7 +49,7 @@ const OcpDashboardBase: React.SFC<OcpDashboardProps> = ({
         })}
       </Grid>
     ) : (
-      <Grid className={css(emptyState)} gutter="md">
+      <Grid gutter="md">
```

Comment:
Similar to AWS, the empty state styles have been removed. Can you please provide a screenshot to confirm this change is desirable?